stdlib/public/Concurrency/CMakeLists.txt

```diff
@@ -32,6 +32,10 @@ if(SWIFT_CONCURRENCY_USES_DISPATCH)
   endif()
 endif()
 
+if(CMAKE_SYSTEM_NAME STREQUAL OpenBSD)
```

Comment:
This is really not the right way to do this.   If other people copy this pattern, we'll end up with a very hard-to-maintain mess.  We're not handling this well today, but let's not make it worse.